apps/client/src/components/RegisterLookupSummary.test.js

```diff
@@ -17,20 +18,22 @@ jest.mock("react-router-dom", () => ({
 afterEach(cleanup);
 
 describe("RegisterLookupSummary", () => {
-  // @TODO Fix the het gebouw test
-  xit("renders correctly in STTR Flow", () => {
+  const WrapperWithContext = (props) => {
     const setActiveState = jest.fn();
-
     const topicMock = "dakraam-plaatsen";
     const topic = findTopicBySlug(topicMock);
 
-    const { queryByText } = render(
-      <RegisterLookupSummary
-        address={addressMock}
-        setActiveState={setActiveState}
-        topic={topic}
-      />
+    return (
+      <Context addressMock={addressMock} topicMock={topic}>
+        <RegisterLookupSummary {...props} setActiveState={setActiveState} />
+      </Context>
     );
+  };
+
+  it("renders correctly in STTR Flow", () => {
+    const topicMock = "dakraam-plaatsen";
+    const topic = findTopicBySlug(topicMock);
```

Comment:
This is defined twice in the test. Would be better to place in the top. Also you can make it `const topic = findTopicBySlug("dakraam-plaatsen");` because `topicMock` is not re-used

Reply:
Never mind. The topic should be different